src/adapter/properties.rs

```diff
@@ -61,6 +61,17 @@ pub(super) fn resolve_item_property<'a>(
     }
 }
 
+pub(super) fn resolve_module_property<'a>(
+    contexts: ContextIterator<'a, Vertex<'a>>,
+    property_name: &str,
+) -> ContextOutcomeIterator<'a, Vertex<'a>, FieldValue> {
+    match property_name {
+        "is_crate" => resolve_property_with(contexts, field_property!(as_module, is_crate)),
+        "is_stripped" => resolve_property_with(contexts, field_property!(as_module, is_stripped)),
```

Comment:
`is_stripped` is another property of the JSON representation, and not of the underlying data model that JSON represents. More JSON elements have `is_stripped`, but I don't remember — do we expose it elsewhere?

It's fine to add, especially if we already expose it in other places. Just pointing it out as another example of the thing I mentioned in my comment above.

Reply:
I see `fields_stripped` on Struct and `variants_stripped` on Enum; I added this by analogy, though TBH I'm not 100% sure.

Reply:
Yup, it's totally fine to leave it then 👍